### PR TITLE
Bulgarian Translation - Fixed

### PR DESCRIPTION
**Pull request by RacerBG**
@levinli303 This one should work.

### CODE REVIEW

File: po3/bg.po [hunk 1]
```diff
@@ -2,21 +2,21 @@
 # This file is distributed under the same license as the celestia package.
 #
 # Translators:
-# Georgi Georgiev <georgiev_1994@abv.bg>, 2020
+# Georgi Georgiev (Жоро) <g.georgiev.shumen@gmail.com>, 2020, 2021
 #
-#, fuzzy
```
Comment (levinli303) [inline po3/bg.po:7]: please do not remove fuzzy

File: po3/bg.po [hunk 2]
```diff
@@ -58,31 +58,31 @@ msgstr "1 000 км/с"
 msgid "10c"
 msgstr "10 c"
 
-# Windows, 10x time speed
+# Mac, Make time go faster
```
Comment (levinli303) [inline po3/bg.po:62]: please do not edit the comment either, this applies to the rest of the PR

**Review by levinli303 — COMMENTED** [review]
I think you should start fresh and edit the raw bg.po

File: po3/bg.po (hunk 2 above)
Comment (RacerBG) [reply po3/bg.po:62]: I did a sync with celestia_ui.pot. These changes were made by Poedit.

Comment (levinli303) [reply po3/bg.po:62]: can you enable editing from maintainers for this PR, I'll use my too to edit it back

Comment (RacerBG) [reply po3/bg.po:62]: Done.

**Comment by levinli303** [comment]
closing since it is corrected in #12

**Pull request closed without merging**